admin/app/controllers/spree/admin/payment_methods_controller.rb

```diff
@@ -26,6 +27,16 @@ def update_turbo_stream_enabled?
       def require_payment_type
         redirect_to spree.admin_payment_methods_path unless params.dig(:payment_method, :type).present?
       end
+
+      def clear_empty_preferences
+        if params[:payment_method].present? && @object.class.const_defined?(:SKIP_BLANK_PREFERENCES_LIST)
```

Comment:
@mad-eel 

we should do it more universal - we should automatically skip blank values for preferences that are already present and are of type `password`

Reply:
@damianlegawiec done